test/lib/spec_linter.js

```diff
@@ -0,0 +1,50 @@
+
+// this is necessary to duplicate the shared functionality of linter.lint
+// within lib/linter.js. there's probably a better way to do this, but to
+// get specs running correctly, this will work in the meantime.
+module.exports = function (linter) {
+
+    return function (config, node) {
+
+        var results = [];
+        var extend = require('lodash.assign');
+
+        // Bail if the linter isn't wanted
+        if (!config[linter.name] || (config[linter.name] && !config[linter.name].enabled)) {
+            return;
+        }
+
+        if (linter.nodeTypes && Array.isArray(linter.nodeTypes) && linter.nodeTypes.length) {
```

Comment:
I think we could remove the `length` check, `indexOf` will return `-1` on an empty array anyway,